packages/amplication-client/src/Workspaces/hooks/workspace.ts

```diff
@@ -0,0 +1,31 @@
+import * as models from "../../models";
+
+export type TData = {
+  currentWorkspace: models.Workspace;
+};
+
+export type TSetData = {
+  setCurrentWorkspace: {
+    token: string;
+  };
+};
+
+export type DType = {
+  createWorkspace: models.Workspace;
+};
```

Comment:
I am not sure what we are using this for, but if the idea is to export it to an external file, I would expect to see more logical names